### ISSUE TITLE
Make a __VERSION__ variable available in the hid module

### ISSUE THREAD

**Issue by squareproton** [trezor/cython-hidapi#31]
Currently there's no way of determining (as far as I can tell) the version of the cython-hidapi installed programatically.

**Comment by prusnak** [comment]
Why is that needed?

**Comment by prusnak** [comment]
Also this should work:

```
>>> from pkg_resources import get_distribution
>>> get_distribution('hidapi').version
'0.7.99.post15'
```

**Comment by squareproton** [comment]
Sorry for the delay in responding to this issue. I missed the notification email somehow.

> Why is that needed?

To aid in debugging.

We recently received some application logs submitted by our users with behavior I couldn't explain or reproduce. The root cause was that our build system had used the wrong version of hidapi and we hadn't noticed before we distributed. Because it took such a long time to figure out so I resolved to add version numbers for our packages to the application logs so if this type of thing happened again it'd be really obvious. While doing this we found all but two packages we used make this available via a `__version__`  variable or something similar.

> from pkg_resources import get_distribution
> get_distribution('hidapi').version

This works in development but we've run into two problems building this into our distributable app because it relies on setuptools. Our windows build process coped with this ok but it included a lot of dependencies we weren't using before and we couldn't get this to work on osx.

In the end we've reverted to 

```
def get_hidapi_version():
    hid_file, regex = hid.__file__, '(hidapi[^\\\/]*)'
    prog = re.compile(regex, re.IGNORECASE)
    matches = prog.search(hid_file)
    if matches:
        return matches.group(0)
    else:
        return hid_file
```
